### DISCUSSION TITLE
"n": 1

### DISCUSSION THREAD

**Discussion by Yumpone** [LostRuins/koboldcpp#369] [Q&A]
I am new to the whole AI world, I hope my question isn't that stupid. 
While going insane, trying to set up temperature and stuff I noticed that ""n":1" never changes.
I tried to find out what it does but failed.
Sooo, I thought somebody might have a clue here.
Thanks for your time in advance! :)

**Comment by LostRuins** [comment]
That parameter is not used in KoboldCpp, it was originally for KoboldAI (main client) used during multi-gen to generate more than 1 response. KoboldCpp does not use it..

**Reply by TheRealPhooney** [reply]
Along the same lines I see "quiet": true and "use_default_badwordsids": true as other parameters, but I can't see any settings that appear to relate directly to them.  Can you shed any light on them or point me to a list of all the possible parameters and what they do?  (most of them I can see what they relate to)

**Reply by LostRuins** [reply]
use_default_badwordsids is new, and is basically giving you the ability to toggle unbantokens from the API.

quiet instructs the main client not to print out generated text in the console.
